src/core/proj/qgscoordinatereferencesystem.cpp

```diff
@@ -2642,24 +2724,27 @@ int QgsCoordinateReferenceSystem::syncDatabase()
 
       QString srsProj4;
       QString srsDesc;
+      QString dbSrsType;
```

Comment:
For the sake of consistency should we rename the other two variables to dbSrs* ?

```suggestion
      QString dbSrsProj4;
      QString dbSrsDesc;
      QString dbSrsType;
```